playbooks/roles/configure-backups/defaults/main.yml

```diff
@@ -0,0 +1,90 @@
+---
+
+backup_enabled: no             # Enable the role
+backup_remove: no               # Set yes for uninstall the role from target system
+backup_cron: yes                # Setup cron tasks for backup
```

Comment:
@sidorovdmitry 
if you mean 'backup_cron: yes' - it's will be used only when backup_enabled